frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -96,6 +96,7 @@ export const StrategyItemContainer: FC<StrategyItemContainerProps> = ({
                 <StyledHeader disabled={Boolean(strategy?.disabled)}>
                     {onDragStart ? (
                         <DragIcon
+                            tabIndex={-1}
```

Comment:
We don't have the capability to let you reorder by keyboard today, so let's remove the drag icon from the tab list. It also has no focus styles, so it's confusing to leave it in.